test/specs/global.deprecations.tests.js

```diff
@@ -1,4 +1,80 @@
 describe('Deprecations', function() {
+	describe('Version 2.7.0', function() {
+		describe('Chart.Controller.update(duration, lazy)', function() {
+			var chart;
+			var addAnimationSpy;
+
+			beforeEach(function() {
+				chart = acquireChart({
+					type: 'doughnut',
+					data: {
+						labels: ['A', 'B', 'C', 'D'],
+						datasets: [{
+							data: [10, 20, 30, 100]
+						}]
+					},
+					options: {
+						cutoutPercentage: 85,
```

Comment:
Both `data` and `cutoutPercentage` were set just to have a "consistent" chart instance, so to speak. Since the type is `doughnut` it made sense to be explicit about the cutout percentage. Will remove them if you think they can cause confusion. 👍